paper.tex

```diff
@@ -44,53 +44,50 @@
     representative Big Data and neuroimaging processing engines, on Dell EMC's 
     Top-500 cluster. 
-    Big Data thresholds were modeled by comparing the data/compute ratio of the 
-    application to the filesystem/concurrent processes ratio. 
+    Big Data thresholds were modeled by comparing the data-write rate of the 
+    application to the filesystem-application concurrent write rate. 
     This model acknowledges the 
 fact that page caching provided by the Linux kernel is critical to the 
 performance of Big Data applications. Results show that in-memory 
 computing alone speeds-up executions by a factor of up to 1.6, whereas 
-when combined with data locality, this factor reaches 5.3. Lazy evaluation strategies
-were found to increase the likelihood of cache hits, further improving processing time. Such important 
+when combined with data locality, this factor reaches 5.3. Lazy evaluation
+strategies
+were found to increase the likelihood of cache hits, further improving
+    processing time. Such important 
 speed-up values are likely to be observed on typical image processing 
-operations performed on images of size larger than 75GB. A ballpark 
-speculation from our model showed that in-memory computing alone will
-not speed-up current 
-functional MRI analyses unless coupled with data 
-locality and processing around 280 subjects concurrently. 
-    In addition, we observe that emulating in-memory computing 
-using in-memory file systems (tmpfs) does not reach the performance of 
-an in-memory engine, presumably due to swapping to disk and the lack of data cleanup.
+operations performed on images of size larger than 75GB. 
 We conclude that Big Data processing strategies are 
 worth developing for neuroimaging 
 applications. 
 \end{abstract}
 
 \section{Introduction} % 1 page with abstract
 
-Big Data processing engines reduce data-related overheads through 
-use of concepts such as data locality, in-memory computing, and lazy evaluation.
-Data locality-aware scheduling, popularized by the MapReduce~\cite{dean2008mapreduce} 
-framework, execute tasks within the nodes which store the data rather than 
-performing costly data transfers across the network. In-memory 
-computing, adopted by MapReduce's successor Apache 
-Spark~\cite{zaharia2016apache}, stores data in memory whenever 
-possible, reducing costly disk I/Os between pipeline steps. 
-Lazy evaluation, also used in Spark, enables further Big Data performance 
-optimizations, by evaluating results only when necessary. Frameworks 
-such as
-MapReduce and Spark have become mainstream tools for data analytics, 
-although many others, such as 
-Dask~\cite{rocklin2015dask}, are emerging. 
+In Big Data applications, there is a significant cost associated with data movement.
+Big Data processing engines have addressed these issues by diminishing the amount 
+of data movement that occurs during the execution of an application. The 
```

Comment:
I'd say 'data overhead' instead of 'data movement' because 'overhead' covers both 'I/O' and actual movement between nodes.